content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.6.
+weight: 20
+---
+
+When you upgrade from Istio 1.5.x to Istio 1.6.x, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.5.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.5.x.
+
+Currently, Istio doesn't support skip-level upgrades. If you are using Istio 1.4, you must upgrade to Istio 1.5 first, and then upgrade to Istio 1.6. If you upgrade from versions earlier than Istio 1.4, you should first disable Galley's configuration validation. 
+
+Updated the Galley deployment:
+
+1. To edit the Galley deployment configuration, run the following command:
+
+{{< text bash >}}
+$ kubectl edit deployment -n istio-system istio-galley
+{{< /text >}}
+
+2. Add the `--enable-validation=false` option to the `command:` section as shown below: 
+
+{{< text yaml >}}
+apiVersion: extensions/v1beta1
+kind: Deployment
+...
+spec:
+...
+  template:
+    ...
+    spec:
+      ...
+      containers:
+      - command:
+        ...
+        - --log_output_level=default:info
+        - --enable-validation=false
+{{< /text >}}
+
+3. Save and quit the editor to update the deployment configuration in the cluster. 
+
+Remove the `ValidatingWebhookConfiguration` Custom Resource (CR) with the following command:
+
+{{< text bash >}}
```

Comment:
@rcaballeromx Seeing the following from the linter:

./en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md:22: MD029 Ordered list item prefix
./en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md:41: MD029 Ordered list item prefix

From Google, it looks like this is because the indentation is restarting and that this should be indented. Spaces don't seem to help. Tabs complain that it should be a multiple of four spaces. Any idea how to get around it?

Reply:
Adding a four space indentation for this text component would be good but it's not what lint is complaining about.
My suggestions above should fix the issue.